crates/bevy_diagnostic/src/system_information_diagnostics_plugin.rs

```diff
@@ -69,23 +69,19 @@ pub mod internal {
         mut sysinfo: Local<Option<System>>,
     ) {
         if sysinfo.is_none() {
-            *sysinfo = Some(System::new_all());
+            *sysinfo = Some(System::new_with_specifics(
+                RefreshKind::new()
+                    .with_cpu(CpuRefreshKind::new().with_cpu_usage())
+                    .with_memory(),
+            ));
         }
         let Some(sys) = sysinfo.as_mut() else {
             return;
         };
 
-        sys.refresh_cpu();
+        sys.refresh_cpu_specifics(CpuRefreshKind::new().with_cpu_usage());
         sys.refresh_memory();
-        let current_cpu_usage = {
-            let mut usage = 0.0;
-            let cpus = sys.cpus();
-            for cpu in cpus {
-                usage += cpu.cpu_usage(); // NOTE: this returns a value from 0.0 to 100.0
-            }
-            // average
-            usage / cpus.len() as f32
-        };
+        let current_cpu_usage = sys.global_cpu_info().cpu_usage();
```

Comment:
No need to go through all CPU to get the average information, it is already available in the `global_cpu_info`.